EventGenerator/src/DecayInOrbitGenerator_module.cc

```diff
@@ -0,0 +1,143 @@
+// This module will be used to make DIO electrons from mu- stopped in any target material
```

Comment:
I don't think we want to have two DIO modules.  Yes it's more work now to add these features to the existing module but it will be less work in the long run.